plugins/core/add_user.go

```diff
@@ -13,7 +13,7 @@ import (
 func NewAddUserCmd(ch cmd.CommandHandler) *cmd.Command {
 	return &cmd.Command{
 		Name:     "add-user",
-		HelpText: "Add a user to a team (admins and tech leads only)",
+		HelpText: "Add a user to a team (admins only)",
```

Comment:
are we removing all tech-lead level permissions?

Reply:
I just took it out for now because I didn't want to write the extra code to check if a given member is a tech lead, since that would require an extra DB query each time we check the user's privilege level. But I was just lazy so I'll add it back in.

Reply:
That would be great!